app/src/androidTest/java/org/mozilla/tv/firefox/ui/PocketBasicUserFlowTest.kt

```diff
@@ -59,6 +60,8 @@ class PocketBasicUserFlowTest {
     fun pocketBasicUserFlowTest() {
         navigationOverlay {
         }.openPocketMegatile {
+        }.returnToOverlay {
```

Comment:
nit: make a comment that we specifically want to test the back button behavior and perhaps had a newline after `returnToOverlay`

I'm concerned someone reading this over will think it's a mistake.